src/Approvals/SFA.DAS.Approvals/InnerApi/Requests/AddDraftApprenticeshipRequest.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+
+namespace SFA.DAS.Approvals.InnerApi.Requests
+{
+    public class AddDraftApprenticeshipRequest
+    {
+        public UserInfo UserInfo { get; set; }
```

Comment:
Also noted that all of these request objects are very similar (I _think_ the same?) - can we use abstraction to reduce code duplication?

Reply:
I've had this argument many times with many people and the counter argument, especially when it's APIM, is around allowing the objects to diverge independently.  Personally, given the amount of copy/paste I've had to do here and previous experience, I think there is definitely something wrong with the overall design of our outer APIs but I've thus far not been able to build a significant enough argument against.

Reply:
Fair enough - I guess this ensures consistency with the rest of the code-base until a decision is (potentially) made otherwise. Thanks for explaining the position.